### PR TITLE
 Update addressfield to 7.x-1.2 (from 7.x-1.1)

### PR DESCRIPTION
**Pull request by deanvucic**
## https://www.drupal.org/project/addressfield/releases/7.x-1.2

Release notes
Changes since 7.x-1.1:

#2507665 by joelpittet: addressfield_get_administrative_areas calls t() nearly 1000 times in one call
#2566773: Wrong CSS class for organisation_block in addressfield_format_organisation_generate
#2578387: CZ and SK have required postal codes
#2570281 by michel.settembrino: Drupal coding standards not followed into addressfield.feeds.inc
#2579129 by skadu: Autocomplete Attribute Typo
Update address formats for Chile and Colombia, based on Google dataset changes.
Remove administrative areas with duplicate keys (defined more than once).
#2528406 by rrfegade: Spelling errors in D7
Stop passing non-english administrative areas to t() in addressfield_get_administrative_areas().
#2546316 by jrb: PHP recoverable fatal error in addressfield when using AJAX
#2530250 by rrfegade: Remove unused varibles from D7
#2421211 by blazey, 7thkey: Administrative Area in Views is rendering key value rather than the rendered string
#2151903 followup: Update addresses.txt for the sub_premise field.
#2510780 by rlhawk: Administrative area for addresses in South Korea should be rendered
#2479205 by marcvangend: Remove "not recommended" from "make all fields optional" and provide real information instead
#2490983 by FMB: Use Lleida and Girona instead of Lérida and Gerona

### CODE REVIEW

File: drupal-org.make [hunk 1]
```diff
@@ -7,14 +7,14 @@ defaults[projects][subdir] = "contrib"
 
 projects[accessible_forms][version] = "1.0-alpha1"
 projects[acquia_connector][version] = "3.1"
-projects[addressfield][version] = "1.1"
+projects[addressfield][version] = "1.2"
```
Comment (seanhamlin) [inline drupal-org.make]: I am for merging in the address module contrib update. Makes sense. Seems to fix bugs.

File: drupal-org.make [hunk 2]
```diff
@@ -7,14 +7,14 @@ defaults[projects][subdir] = "contrib"
...
 projects[admin_views][version] = "1.6"
 projects[agls][version] = "1.0-beta3"
 projects[bean][version] = "1.11"
 projects[bean_panels][version] = "1.3"
 projects[better_exposed_filters][version] = "3.4"
 projects[bigmenu][version] = "1.3"
-projects[breakpoints][version] = "1.3"
+projects[breakpoints][version] = "1.4"
```
Comment (seanhamlin) [inline drupal-org.make]: You have added the breakpoint module update in this PR too, can you remove that (and if you need it, raise another PR).

Squashing your commits would be nice too (but can be done at PR merge time).

**Review by seanhamlin — CHANGES_REQUESTED** [review]

File: drupal-org.make (hunk 2 above)
Comment (deanvucic) [reply drupal-org.make]: Done. Thanks Sean.

**Review by seanhamlin — APPROVED** [review]

**Comment by seanhamlin** [comment]
Cool. Assigned @tobybellwood to see if this can be merged.

**Pull request closed without merging**